packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -0,0 +1,344 @@
+        selectors.searchBoxSubmitBtn
+      );
+      const clearIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxClearIcon
+      );
+
+      expect(input).not.toBeNull();
+      expect(input.placeholder).toEqual(defaultOptions.placeholder);
+      expect(submitButton).not.toBeNull();
+      expect(clearIcon).toBeNull();
+    });
+  });
+
+  describe('when the textarea property is set to true', () => {
+    it('should display the search textarea properly', async () => {
+      const element = createTestComponent({...defaultOptions, textarea: true});
+      await flushPromises();
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+      const clearIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxClearIcon
+      );
+      const textarea = element.shadowRoot.querySelector(
+        selectors.searchBoxTextArea
+      );
+
+      expect(textarea).not.toBeNull();
+      expect(textarea.placeholder).toEqual(defaultOptions.placeholder);
+      expect(submitButton).not.toBeNull();
+      expect(clearIcon).toBeNull();
+    });
+  });
+
+  describe('when the withoutSubmitButton is set to false', () => {
+    it('should not display the searchIcon to the left of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: false,
+      });
+      await flushPromises();
+
+      const searchIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxSearchIcon
+      );
+
+      expect(searchIcon).toBeNull();
+    });
+
+    it('should display the submit button to the right of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: false,
+      });
+      await flushPromises();
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+
+      expect(submitButton).not.toBeNull();
+    });
+  });
+
+  describe('when the withoutSubmitButton is set to true', () => {
+    it('should display the searchIcon to the left of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: true,
+      });
+      await flushPromises();
+
+      const searchIcon = element.shadowRoot.querySelector(
+        selectors.searchBoxSearchIcon
+      );
+
+      expect(searchIcon).not.toBeNull();
+      expect(searchIcon.classList.contains('slds-input__icon_left')).toBe(true);
+    });
+
+    it('should not display the submit button to the right of the searchbox', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        withoutSubmitButton: true,
+      });
+      await flushPromises();
+
+      const submitButton = element.shadowRoot.querySelector(
+        selectors.searchBoxSubmitBtn
+      );
+
+      expect(submitButton).toBeNull();
+    });
+  });
```

Comment:
```suggestion
  describe('when the withoutSubmitButton is set to true', () => {
    it('should not display the submit button to the right of the searchbox', async () => {
      const element = createTestComponent({
        ...defaultOptions,
        withoutSubmitButton: true,
      });
      await flushPromises();

      const searchIcon = element.shadowRoot.querySelector(
        selectors.searchBoxSearchIcon
      );

      const submitButton = element.shadowRoot.querySelector(
        selectors.searchBoxSubmitBtn
      );

      expect(searchIcon).not.toBeNull();
      expect(submitButton).toBeNull();
    });
  });
```
Same as above. I would merge these two tests